src/Tools/IdeCoreBenchmarks/SerializationBenchmarks.cs

```diff
@@ -13,6 +13,8 @@
 using Microsoft.CodeAnalysis.CSharp.Syntax;
 using Microsoft.CodeAnalysis.Text;
 
+#pragma warning disable CS0618 // Type or member is obsolete
```

Comment:
For these you could also the class itself as [Obsolete] which might make it clearer this obsolete in it's own right.